### ISSUE TITLE
About Focal Loss Weight (balancing parameter)

### ISSUE THREAD

**Issue by kyuewang17** [MIV-XJTU/ARTrack#29]
Hello, I was looking your amazing work and happened to find something strange about the loss balancing parameters.

In script, [artrack_seq.py](lib/train/actors/artrack_seq.py).
Line #213 seems to assign loss balancing weight for the Cross-entropy Loss. However, the value is set to 0.

In order to reproduce your work, might the value supposed to be changed to 1, since the GIoU weight is already set to 2.0 in the [artrack_seq_256_full.yaml](experiments/artrack_seq/artrack_seq_256_full.yaml) script?

**Comment by AlexDotHam** [comment]
When training the second stage, the loss weight of CE is zero, based on an intuition that the first stage has trained a sufficiently good vocabulary. Moreover, GIoU loss brings randomness in training and robustness of model prediction, in fact, CE loss is the same as L1 loss in traditional tracking, which is not good enough for complex scenarios with unclear target boundaries and fuzzy manual annotations.

**Comment by kyuewang17** [comment]
Oh, thanks for the information. I might have missed that on the paper... Could you point out the part that explains the 2nd-stage training, in the paper? I would like to refer it and then read the code again:)
